Language/Functions/USB/Keyboard.adoc

```diff
@@ -24,6 +24,31 @@ The library supports the use of modifier keys. Modifier keys change the behavior
 --
 // OVERVIEW SECTION ENDS
 
+[float]
+=== Compatible Hardware
+HID is supported on the following boards:
+[options="header"]
+|=========================================================
+| Board                | Pins                     
```

Comment:
Looks good. I think, let's not link all MKR boards, but instead link to the family, like this: `https://docs.arduino.cc/#mkr-family`.